services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/HankeServiceITests.kt

```diff
@@ -114,8 +116,7 @@ class HankeServiceITests : DatabaseTest() {
 
     @Test
     fun `create Hanke with full data set succeeds and returns a new domain object with the correct values`() {
-        val hanke: Hanke =
-            getATestHanke().withYhteystiedot { it.id = null }.withPerustaja().withHankealue()
+        val hanke: Hanke = getATestHanke().withYhteystiedot { it.id = null }.withHankealue()
```

Comment:
There should be a test with the perustaja data. Readding it to the test data here causes a failure at
```
assertThat(hankeKayttajaRepository.findAll()).hasSize(4)
```

Reply:
There is now two tests. One including perustaja and one with no perustaja and no contacts.